pkg/kubectl/cmd/apply.go

```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/golang/glog"
 	"github.com/jonboulle/clockwork"
 	"github.com/spf13/cobra"
+	"k8s.io/kubernetes/pkg/kubectl/polymorphichelpers"
```

Comment:
`goimports` the imported packages or manually sort.

Reply:
Thanks, done